ozaria/site/components/interactive/PageInteractiveEditor/index.vue

```diff
@@ -59,14 +59,17 @@
         try {
           this.interactive = new Interactive(await getInteractive(slug))
         } catch (e) {
-          noty({ text: `Error finding slug '${slug}'.`, type: 'error', timeout: '1000' })
+          noty({ text: `Error finding slug '${slug}'.`, type: 'error', timeout: 2000 })
           return this.fetchList()
         }
         const data = $.extend(true, {}, this.interactive.attributes)
         const el = $(`<div></div>`)
         const treemaOptions = {
           data: data,
           schema: Interactive.schema,
+          // Automatically uploads the file to /file/interactives/<fileName>
+          // You can view files at /admin/files
+          filePath: 'interactives',
           callbacks: {
```

Comment:
Nitpick. Is there a reason to use interactives in plural instead of interactive?

Reply:
Great pickup, I could not use singular `interactive` because an api is called from `FilesComponent.vue` to display files on /admin/files page, which convert `interactive` to an object id 😮 (https://github.com/codecombat/codecombat/pull/5399/files#diff-c21e1a02d26e62dfce2f704de35dd4f4R9)
Hence using the plural name for file location.